solr/test-framework/src/java/org/apache/solr/SolrTestCaseHS.java

```diff
@@ -107,72 +106,84 @@ public static ModifiableSolrParams params(SolrParams params, String... moreParam
     }
     return result;
   }
-  
-  
+
   @SuppressWarnings({"unchecked"})
-  public static Object createDocObjects(@SuppressWarnings({"rawtypes"})Map<Comparable, Doc> fullModel,
-                                        @SuppressWarnings({"rawtypes"})Comparator sort, int rows,
-                                        Collection<String> fieldNames) {
+  public static Object createDocObjects(
+      @SuppressWarnings({"rawtypes"}) Map<Comparable, Doc> fullModel,
+      @SuppressWarnings({"rawtypes"}) Comparator sort,
+      int rows,
+      Collection<String> fieldNames) {
     List<Doc> docList = new ArrayList<>(fullModel.values());
     Collections.sort(docList, sort);
     @SuppressWarnings({"rawtypes"})
     List sortedDocs = new ArrayList(rows);
     for (Doc doc : docList) {
       if (sortedDocs.size() >= rows) break;
-      Map<String,Object> odoc = toObject(doc, h.getCore().getLatestSchema(), fieldNames);
+      Map<String, Object> odoc = toObject(doc, h.getCore().getLatestSchema(), fieldNames);
       sortedDocs.add(toObject(doc, h.getCore().getLatestSchema(), fieldNames));
     }
     return sortedDocs;
   }
 
-
-  public static void compare(SolrQueryRequest req, String path, Object model,
-                             @SuppressWarnings({"rawtypes"})Map<Comparable, Doc> fullModel) throws Exception {
+  public static void compare(
+      SolrQueryRequest req,
+      String path,
+      Object model,
+      @SuppressWarnings({"rawtypes"}) Map<Comparable, Doc> fullModel)
+      throws Exception {
     String strResponse = h.query(req);
 
     Object realResponse = ObjectBuilder.fromJSON(strResponse);
     String err = JSONTestUtil.matchObj(path, realResponse, model);
     if (err != null) {
-      log.error("RESPONSE MISMATCH: {}\n\trequest={}\n\tresult={}" +
-          "\n\texpected={}\n\tmodel={}"
-          , err, req, strResponse, JSONUtil.toJSON(model), fullModel);
+      log.error(
+          "RESPONSE MISMATCH: {}\n\trequest={}\n\tresult={}" + "\n\texpected={}\n\tmodel={}",
```

Comment:
I bet if you declare the first arg as a String in the previous line (`var msg = "RESPONSE ......."`) then it would then the following log.error would be one line.